notebooks/peerscout/peerscout-get-editor-pubmed-paper-ids.ipynb

```diff
@@ -77,102 +75,124 @@
     "    prefix=output_table_prefix,\n",
     "    suffix='editor_pubmed_ids'\n",
     ")"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "read_big_query = partial(_read_big_query, project_id=project_id)"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "default_query_props = dict(project=project_id, dataset=source_dataset)"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "_sql = (\n",
-    "    'SELECT person_id FROM `{table}`'\n",
-    ").format(table=editor_parsed_pubmed_ids_table_name)\n",
+    "    '''\n",
+    "    SELECT\n",
+    "        person_id,\n",
+    "        pubmed_url,\n",
+    "        provenance.imported_timestamp as imported_timestamp\n",
+    "    FROM `{table}`\n",
```

Comment:
Another thought that came to mind is that the table here will probably need to be the view that you were going to discuss. (Because we want to compare it to the latest version)

Reply:
Just mentioning that before merging this PR, we will need to make the following steps work with the changed output table.
If we are using a (sort-of) "materialized view" for that, then you could use that here too.

Reply:
Sure